src/test/java/test/dependent/DependentTest.java

```diff
@@ -159,7 +159,7 @@ public void methodDependencyBetweenClassesShouldWork(Class[] classes, boolean pr
       {GitHub1380Sample.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}, true},
       {GitHub1380Sample2.class, new String[] {"testMethodC", "testMethodB", "testMethodA"}, true},
       {GitHub1380Sample3.class, new String[] {"testMethodA", "testMethodB", "testMethodC"}, true},
-      {GitHub1380Sample4.class, new String[] {"testMethodB", "testMethodA", "testMethodC"}, true}
+      {GitHub1380Sample4.class, new String[] {"testMethodB", "testMethodC", "testMethodA"}, true}
```

Comment:
The change to this test is because `testMethodC` doesn't have any dependencies, `testMethodA` does, and this is running parallel. In terms of time after the test run starts, `testMethodA` will be run at the exact same time (it will be the next method run after `testMethodB`. However, because this is running in parallel and `testMethodC` has no dependencies, there's no reason it can't run in parallel to `testMethodB`.